controllers/account/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -13,7 +13,8 @@ spec:
         securityContext:
           runAsNonRoot: true
           allowPrivilegeEscalation: false
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.11.0
+        image: gcr.dockerproxy.com/kubebuilder/kube-rbac-proxy:v0.11.0
```

Comment:
Commit code do not change image path?

Reply:
changed it，It was originally just for the convenience of testing on linux